cli/cmd/policy.go

```diff
@@ -124,6 +124,39 @@ To view the LQL query associated with the policy, use the query id shown.
 		Args:    cobra.ExactArgs(1),
 		RunE:    showPolicy,
 	}
+
+	// This is an experimental command.
+	// policyDisableTagCmd represents the policy disable-policy command
+	policyDisableTagCmd = &cobra.Command{
+		Use:     "disable-policy [policy_id]",
```

Comment:
nit. should this just be `disable` . i.e. `lacework policy disable` vs. `lacework policy disable-policy` ?  Same comment for `enable`.

Reply:
👍